packages/web/src/react-query/governance/use-get-my-delegation.ts

```diff
@@ -0,0 +1,26 @@
+import { useQuery, UseQueryOptions } from "@tanstack/react-query";
+
+import { useGnoswapContext } from "@hooks/common/use-gnoswap-context";
+import { GetMyDeligationRequest, MyDelegationInfo } from "@repositories/governance";
+
+import { QUERY_KEY } from "../query-keys";
+
+const REFETCH_INTERVAL = 10_000;
```

Comment:
@dongwon8247 @roje1 
Will there be a refetch interval to retrieve delegation information?

Reply:
It's not important right now.
This can be fixed later by considering the refetch policy again.

Reply:
yepp, this PR is under develop and no side effect for current product.
It's good to update later.